applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.h

```diff
@@ -199,6 +204,20 @@ int broadcast_source_start(uint8_t big_index, struct bt_le_ext_adv *ext_adv);
  */
 int broadcast_source_stop(uint8_t big_index);
 
+/**
+ * @brief	Get the broadcast ID for the given Broadcast Isochronous Group (BIG).
+ *
+ * @note	The broadcast ID is a 24-bit value that is used to identify the broadcast. This
```

Comment:
Some docs refer to octets and another to bits.

Reply:
Can change all to be octets,